contracts/vaults/TreasuryVault/RibbonAutocallVault.sol

```diff
@@ -123,6 +123,7 @@ contract RibbonAutocallVault is RibbonTreasuryVaultLite, AutocallVaultStorage {
         nObsFreq = _obsFreq;
         nPeriod = period;
         autocallSeller = _autocallSeller;
```

Comment:
can remove `autocallSeller = _autocallSeller;` now

Reply:
it was decided to be keep this line due to the possible "ERC20: transfer to the zero address" error occurring when rolling to next round on the first round | in the case someone made a USDC transfer to the contract